prog/dftb+/lib_extlibs/slatec.F90

```diff
@@ -0,0 +1,165 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2019  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Interfaces for the SLATEC routines needed in DFTB+
+module dftbp_slatec
+  use dftbp_accuracy, only : rdp
+  implicit none
+  private
+
+  public :: zbesi, dgamit, dgamic
+
+
+  interface
+
+    !***BEGIN PROLOGUE  ZBESI
```

Comment:
`!*` can interfer with the FORD processing, usually leading to strange error messages.